docs/content/dagster-cloud/managing-deployments/dagster-cloud-yaml.mdx

```diff
@@ -0,0 +1,143 @@
+| `locations.code_source.package_name` | Use if your dagster code is structured as a package                                   | string (folder name) |
+| `locations.code_source.python_file`  | Use if your dagster code is in a single file                                          | string (.py file)    |
+| `locations.code_source.module_name`  | The module that contains your dagster code (which matches pyproject.toml)                                            | string (module name) |
+
+Example for a single code location:
+
+```yaml
+#dagster_cloud.yaml
+
+locations:
+  - location_name: data-eng-pipeline
+    code_source:
+      package_name: quickstart_etl
+```
+
+Example for multiple code locations:
+
+```yaml
+#dagster_cloud.yaml
+
+locations:
+  - location_name: data-eng-pipeline
+    code_source:
+      package_name: quickstart_etl
+  - location_name: machine_learning
+    code_source:
+      python_file: ml/ml_model.py
+```
+
+## Configuring your working directory
+
+There might be some cases where you want to run your Dagster code from a different directory than the root of your code repository. You can specify the directory you want to run your Dagster code from in the dagster_cloud.yaml file using the `working_directory` option.
+
+Parameter reference table:
+
+| Parameter           | Description                                            | Data type     |
+| ------------------- | ------------------------------------------------------ | ------------- |
+| `working_directory` | The path of the directory that Dagster should run from | string (path) |
+
+```yaml
+#dagster_cloud.yaml
+
+locations:
+  - location_name: data-eng-pipeline
+    code_source:
+      package_name: quickstart_etl
+    working_directory: ./project_directory
+```
+
+## Build directory
+
+Setting a build directory is useful if your setup.py or requirements.txt is not at your project root but in a subdirectory. This is common if you have multiple python packages within a single dagster project.
+
+The build registry is applicable to hybrid deployments only.
+
+Parameter reference table:
+
+| Parameter                   | Description                                                                                                                                                       | Data type                |
+| --------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------ |
+| `locations.build.directory` | The directory in your project code that you want to deploy, the default is `.`. If you have sub-folders you can specific the path to only deploy that sub-project | string (path)            |
+| `locations.build.registry`  | The Docker registry to push your code location to in the registry (Hybrid only)                                                                                   | string (docker registry) |
```

Comment:
```suggestion
| `locations.build.registry`  | The Docker registry to push your code location to.                                                                              | string |
```

Removing `(Hybrid only)` as you stated before this that this entire section is applicable only to Hybrid.

I'm also suggesting we remove `in the registry` because it felt redundant, but if it changes the meaning of the sentence, ignore that part. My thinking is this already says we're pushing it to a Docker registry, so 'in the registry' was duplicative.